recipes/snpeff/5.1d/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set snpeff_ver = "v5_1d" %}
+# NOTE: if the version contains a trailing letter, use the <d>.<d>.1<l> format
+# Please update recipes/snpsift as well!
+{% set version = "5.1d" %}
+{% set sha256 = "919e0595c08e86d1dd82279723c83cb872070244ee4ce0cb3167bde2b272893b" %}
+
+about:
+  home: 'http://snpeff.sourceforge.net/'
+  license: "LGPLv3"
+  summary: "Genetic variant annotation and effect prediction toolbox"
+
+package:
+  name: snpeff
+  version: {{ version }}
+
+build:
+  number: 1
+  noarch: generic
+  run_exports:
+    - {{ pin_compatible('snpeff', max_pin="x.x") }}
```

Comment:
Again, same story here.

Reply:
That looks good to me and is the same as the latest version.